### PR TITLE
Fix parsing non crypted messages

### PR DESCRIPTION
**Pull request by lukas1**

### CODE REVIEW

File: Tests/DCoreKitTests/MessagingApiTests.swift [hunk 1]
```diff
@@ -86,6 +90,6 @@ class MessagingApiTests: XCTestCase {
         ("testGetAllResponsesByReceiverUsingWss", testGetAllResponsesByReceiverUsingWss),
         ("testGetAllResponsesBySenderUsingWss", testGetAllResponsesBySenderUsingWss),
         ("testGetAllDecryptedMessagesBySenderUsingWss", testGetAllDecryptedMessagesBySenderUsingWss),
-        ("testGetAllDecryptedMessagesBySenderUsingWrongCredentialsUsingWss", testGetAllDecryptedMessagesBySenderUsingWrongCredentialsUsingWss),
+        ("testZLastGetAllDecryptedMessagesBySenderUsingWrongCredentialsUsingWss", testZLastGetAllDecryptedMessagesBySenderUsingWrongCredentialsUsingWss),
```
Comment (fl3xman) [inline Tests/DCoreKitTests/MessagingApiTests.swift]: typo ZLast

File: Sources/DCoreKit/Model/Message.swift [hunk 2]
```diff
@@ -104,3 +104,9 @@ public struct MessagePayloadReceiver: Codable {
         nonce
     }
 }
+
+private extension Optional where Wrapped == Address {
+    func isEmptyAddress() -> Bool {
+        return isNil() || self?.publicKey.data.allSatisfy { $0 == 0x00 } ?? true
```
Comment (fl3xman) [inline Sources/DCoreKit/Model/Message.swift]: this check self?.publicKey.data.allSatisfy { $0 == 0x00 } can be in publickKey as getter

**Review by fl3xman — CHANGES_REQUESTED** [review]

File: Tests/DCoreKitTests/MessagingApiTests.swift (hunk 1 above)
Comment (lukas1) [reply Tests/DCoreKitTests/MessagingApiTests.swift]: it's not a typo, it's intentional. Tests are run based on alphabetical ordering, so I wanted to make sure that this test will run last

Comment (fl3xman) [reply Tests/DCoreKitTests/MessagingApiTests.swift]: Order should not matter? what if test running behaviour changes from alpha-numeric to fully random? I understand u need some data before hand, can u add those data into test?

Comment (lukas1) [reply Tests/DCoreKitTests/MessagingApiTests.swift]: all tests in for messaging api have been modified to work regardless of order of execution

File: Sources/DCoreKit/Model/Message.swift (hunk 2 above)
Comment (lukas1) [reply Sources/DCoreKit/Model/Message.swift]: done

**Review by fl3xman — APPROVED** [review]

**Pull request merged**